.azure/scripts/test.yml

```diff
@@ -12,8 +12,8 @@ steps:
     version: '$(jdk.version)'
 
 - script: |
-    python -m pip install --upgrade pytest setuptools
-    python setup.py build_ext --inplace
+    python -m pip install setuptools
+    python setup.py develop
```

Comment:
Initially I had this as `python -m pip install --editable .`. It seems this is causing the issue seen with pytest.